packages/ketcher-standalone/src/infrastructure/services/struct/indigoWorker.ts

```diff
@@ -109,8 +109,15 @@ self.onmessage = (e: MessageEvent<InputMessage<CommandData>>) => {
     case Command.Layout: {
       const data: LayoutCommandData = message.data as LayoutCommandData;
       handle(
-        (indigo, indigoOptions) =>
-          indigo.layout(data.struct, data.format, indigoOptions),
+        (indigo, indigoOptions) => {
+          const response = indigo.layout(
+            data.struct,
+            data.format,
+            indigoOptions,
+          );
+          const { struct } = JSON.parse(response);
```

Comment:
Why have you decided to omit other values from layout? I find it very risky

Reply:
Previously layout return just structure.
Now we set output-content-type = 'application/json'  in payload options. 
 If set to 'application/json' - function will return json with next format:
{
"struct": output_molecule,
"format": output_format,
"original_format": original_format,
}
So I believe it is expected.